### PR TITLE
Homepage [SC-257]

### PR DESCRIPTION
**Pull request by RVMendoza**

### CODE REVIEW

**Comment by RVMendoza** [comment]
image

**Comment by RVMendoza** [comment]
@VinSpee I would say ui wise, this is ready for code review.

**Comment by RVMendoza** [comment]
This relates to https://github.com/LevelEleven/scorecard/issues/84

File: app/scripts/app.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
-/*Cannot be found...I don't know what do so I'm commenting it out*/
-require('svg-shim');
+var SVGshim       = require('svg-shim');
```
Comment (VinSpee) [inline app/scripts/app.jsx]: I don't think you need this here.

File: app/scripts/app.jsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
+var SVG4everybody = require('./svg4everybody');
+var SVG           = require('svg');
```
Comment (VinSpee) [inline app/scripts/app.jsx]: I don't think you need this here

File: app/scripts/components/menu-bar.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    return (
+      <div role="banner" className="Arrange Arrange--middle u-marginBm">
+        <div className="Arrange-sizeFit">
+          <a href="#/" className="Icon Icon--medium MenuBarIcon u-alignMiddle u-block">
+            <SVG icon="home" />
+          </a>
+        </div>
+        <div className="Arrange-sizeFill">
+          <div className="MenuBarTitle u-textCenter">
+            <span className="Text--headline u-marginRb">Scorecard</span><span className="Text--body">builder</span>
```
Comment (VinSpee) [inline app/scripts/components/menu-bar.jsx]: clean up this markup (new lines)

File: app/scripts/components/menu-bar.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+var React = require('react');
+var SVG   = require('svg');
+
```
Comment (VinSpee) [inline app/scripts/components/menu-bar.jsx]: might need the SVGshim here

File: app/scripts/views/home.jsx [hunk 5]
```diff
@@ -0,0 +1,57 @@
+var React = require('react');
+var SVG   = require('svg');
```
Comment (VinSpee) [inline app/scripts/views/home.jsx]: might need SVGshim

File: app/styles/brand/lib/brand.css [hunk 6]
```diff
@@ -0,0 +1,12 @@
+/** @define Brand; use strict */
+/*
+ * Branding elements
+ */
+
+.BrandHeadline {
```
Comment (VinSpee) [inline app/styles/brand/lib/brand.css]: private tree elements should be defined with a single dash like so:

```
.Brand-headline {}
```

File: app/styles/definition/lib/definition.css [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+ */
+
+/*
+For some reason defintion lists and its items
+like to forcibly be on their own line.
+Lea Verou has a fix:
+http://lea.verou.me/2012/02/flexible-multiline-definition-lists-with-2-lines-of-css/
+*/
+
+.DefitionTerm, .DefinitionDefinition, dt, dd {
```
Comment (VinSpee) [inline app/styles/definition/lib/definition.css]: private tree elements should be defined with a single dash like so:

```
.Definition-term {}
```

File: app/styles/menubar/lib/menubar.css [hunk 8]
```diff
@@ -0,0 +1,14 @@
+/** @define Menubar; use strict */
+/*
+ * Top action bar
+ */
+
+.MenuBarTitle {
```
Comment (VinSpee) [inline app/styles/menubar/lib/menubar.css]: private tree elements should be defined with a single dash

File: app/styles/slat/lib/slat.css [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+
+.Slat {
+  padding: var(--s-4);;
+  border: 2px solid var(--white);
+  border-radius: var(--borderRadius);
+  font-size: var(--s1);
+  transition: var(--transitionAnimation);
+}
+
+.Slat:hover, .Slat:active {
```
Comment (VinSpee) [inline app/styles/slat/lib/slat.css]: You should also give a class to represent these states: `.Slat.isFocused`, `.Slat.isActive`

Comment (RVMendoza) [inline app/styles/slat/lib/slat.css]: why do they need this for pseudo elements?

**Comment by RVMendoza** [comment]
Whoops, thought I split those up in more commits. Made those changes @VinSpee, thanks

**Pull request merged**